.travis.yml

```diff
@@ -8,8 +8,12 @@ android:
   components:
     - platform-tools
     - tools
-    - build-tools-25.0.0
+    - build-tools-25.0.2
     - android-25
+
+    # RS components
+    - build-tools-23.02
+    - android-23
```

Comment:
No longer needed.  RS should always be the same as BASdk, so I'd just go ahead and update RS to 25.0.2.  It should be at 25 currently.